docs/ota-client-guide/modules/ROOT/pages/upload-large-binary.adoc

```diff
@@ -28,15 +28,15 @@ garage-sign init \
 ----
 +
 
-. Pull the latest version of the `targets.json` file to your local repository.
+. Pull the latest version of the `targets.json` file.
 +
 [source,bash]
 ----
 garage-sign targets pull \
   --repo <localreponame>
 ----
 
-. Upload your binary file to your local repository and specify the name and version of your `targets.json` file.
+. To upload your binary to OTA Connect, specify its name and version.
 +
 TIP: You can find the `name` and `version` fields in the `targets.json` file under the `custom` field, as shown in the xref:customise-targets-metadata.adoc#_anatomy_of_targets_json_metadata[example].

```

Comment:
```suggestion
```
Let's get rid of this tip here. It doesn't make a lot of sense, because you can't find the name and version there yet--this is still one step before that metadata is added.

Reply:
Deleted